### ISSUE TITLE
qwen2.5-vl支持仅文本输入吗？

### ISSUE THREAD

**Issue by QN-time** [QwenLM/Qwen2.5-VL#951]
我不太清楚qwen2.5-vl是否能支持仅文本输入
我尝试去除video和image，但似乎会报错
            input_prompt = hf_tokenizer.apply_chat_template(
                messages, tokenize=False, add_generation_prompt=True
            )
            input_ids = hf_tokenizer(
                input_prompt, return_tensors="pt", padding=True, truncation=True
            ).to("cuda")

**Comment by JJJYmmm** [comment]
You can refer to this code.
```python
import torch
from transformers import AutoTokenizer, AutoProcessor, AutoModelForVision2Seq

if __name__ == '__main__':
    model_path = "Qwen/Qwen2.5-VL-7B-Instruct"
    text_tokenizer = AutoTokenizer.from_pretrained(model_path, use_fast = True)
    processor = AutoProcessor.from_pretrained(model_path, use_fast = True)
    model = AutoModelForVision2Seq.from_pretrained(model_path, 
                                    torch_dtype = torch.bfloat16,
                                    attn_implementation = "flash_attention_2",
                                    device_map = "cpu").cuda()

    messages = [
        {"role": "system", "content": "You are a helpful assistant."},
        {"role": "user", "content": "Hello! Who are you?"},
    ]

    # solution 1. use the processor to tokenize the text
    # text = processor.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
    # inputs = processor(text=[text], text_kwargs = {"padding": False}, return_tensors="pt")

    # solution 2. use the tokenizer to tokenize the text
    text = text_tokenizer.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
    inputs = text_tokenizer(text, return_tensors="pt", padding=False)

    inputs = inputs.to(model.device)

    output_ids = model.generate(**inputs, max_new_tokens=512)
    generated_ids = [output_ids[len(input_ids):] for input_ids, output_ids in zip(inputs.input_ids, output_ids)]
    output_text = processor.batch_decode(generated_ids, skip_special_tokens=True, clean_up_tokenization_spaces=True)

    print(output_text[0])
```

**Comment by QN-time** [comment]
Thanks, this solved my problem very well！
